nixos/modules/virtualisation/anbox.nix

```diff
@@ -73,9 +73,6 @@ in
 
     environment.systemPackages = with pkgs; [ anbox ];
 
-    boot.kernelModules = [ "ashmem_linux" "binder_linux" ];
```

Comment:
We still want the modules to be loaded I think? I don't use anbox anymore.

Reply:
There is no "modules" for anbox with the kernel config that got added, they're built-in to kernel now

Reply:
Yeah, that's what I gathered from looking at other PRs related to this as well.

Reply:
I know but they still need to be loaded before anbox can work. I don't think they're loaded automatically unless we declare NixOS to do so.

Reply:
Hopefully an anbox user can support with that.